pallets/asset/src/lib.rs

```diff
@@ -426,6 +436,13 @@ decl_storage! {
                 };
                 ClassicTickers::insert(&import.ticker, classic_ticker);
             }
+
+            let sys_did = &config.reserved_country_currency_codes[0].0;
+            for code in &config.reserved_country_currency_codes[0].1 { 
+                let expiry = <Module<T>>::ticker_registration_checks(&code, *sys_did, true, config);
```

Comment:
`config` doesn't seem like the right type? You're supposed to pass in a `TickerRegistrationConfig` here.